Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -1646,6 +1680,9 @@ Lastly, see if you can play with the version number of a man-in-the-middled app
 - MSTG-PLATFORM-7: "If native methods of the app are exposed to a WebView, verify that the WebView only renders JavaScript contained within the app package."
 - MSTG-PLATFORM-8: "Object serialization, if any, is implemented using safe serialization APIs."
 - MSTG-PLATFORM-9: "The app does not allow any other application to overlay on itself."
+- MSTG-PLATFORM-10: "A WebView's cache, storage, and loaded resources (JavaScript, etc.) should be cleared before the WebView is destroyed."
```

Comment:
We are only adding the requirements from MASVS if the test case is also added here. This one is not yet outlined here, so please remove it for now.